### DISCUSSION TITLE
docs/3.x/api-usage/slug-fields

### DISCUSSION THREAD

**Discussion by giscus** [saleor/saleor-docs#799] [General]
# docs/3.x/api-usage/slug-fields

A slug is a short label representing an object or page's content. The main reason for using a slug is the ability to incorporate it into URLs. Translated names can contain all UTF-8 signs, allowing for URLs to be written by people with different keyboard layouts.

https://docs.saleor.io/docs/3.x/api-usage/slug-fields

**Comment by delemeator** [comment]
What is the rational behind making slugs not translatable? Wouldn't it be beneficial for SEO to localize them? Do you have any plans to change this in the future?
